HW02_B_Mimic_starter.Rmd

```diff
@@ -91,7 +106,20 @@ The above graph lets you see some colobrlind friendly palettes. For the graph be
 Now using the above mpg dataset, make this graph
 
 ```{r graph 4 code}
-
+mpg %>% 
+  ggplot(aes(x = class, y = cty)) +
+  geom_jitter(aes(fill = class, color = class), height = 0) + #This specific jitter will only allow for the vertical movement of the dots, mimicing what is seen in the example

```

Comment:
Thanks for this! I didn't realize the different jitter parameters that can be set.